### PR TITLE
Update readme adding image size

### PR DESCRIPTION
**Pull request by Forbu**

### CODE REVIEW

**Comment by anibali** [comment]
Nice idea, but the information shown is outdated. The badges are showing the version and size for `1.8.1-cuda11.1`, but the image currently tagged as latest is actually `1.10.0-cuda11.3`.

**Comment by Forbu** [comment]
I will solve this ^^

**Comment by Forbu** [comment]
It's good I think

File: README.md [hunk 1]
```diff
@@ -1,6 +1,9 @@
 ## PyTorch Docker image
 
-[![Docker Automated build](https://img.shields.io/docker/automated/anibali/pytorch.svg)](https://hub.docker.com/r/anibali/pytorch/)
+[![Docker Automated build](https://img.shields.io/docker/automated/anibali/pytorch)](https://hub.docker.com/r/anibali/pytorch/)
+[![Docker Automated build](https://img.shields.io/docker/image-size/anibali/pytorch)](https://hub.docker.com/r/anibali/pytorch/)
```
Comment (anibali) [inline README.md]: This also needs to be changed: `https://img.shields.io/docker/image-size/anibali/pytorch/latest`

**Comment by anibali** [comment]
Thanks!

**Pull request merged**